playwright/tests/test_17_EstTool_UI_AssumpLimitations.py

```diff
@@ -92,7 +92,7 @@ def test_example(page: Page) -> None:
     expect(page.get_by_text("Clearly there is a large")).to_be_visible()
     expect(page.get_by_text("The estimated kWh of cloud")).to_be_visible()
     expect(page.get_by_role("heading", name="Downstream Emissions")).to_be_visible()
-    expect(page.get_by_text("At present we focus on the")).to_be_visible()
+    expect(page.get_by_text("To do this we have collated")).to_be_visible()
```

Comment:
Now that the panels are hidden when not selected there are 2 elements in the DOM with the old text so I've used the start of the second sentence in the section instead.